elements/pl-file-editor/pl-file-editor.js

```diff
@@ -44,6 +44,12 @@ window.PLFileEditor = function(uuid, options) {
         this.editor.setOption('maxLines', Infinity);
     }
 
+    if (options.plOptionFocus) {
+        this.plOptionFocus = true;
+    } else {
+        this.plOptionFocus = false;
+    }
```

Comment:
```suggestion
    this.plOptionFocus = !!options.plOptionFocus;
```

The double-bang forces the variable to a boolean, but it's not actually needed here as far as I can see.

Reply:
It may not be needed (I was trying to be explicit while debugging). Should we just take it out?

Reply:
If it's not needed then lets take it out

Reply:
Okay, I took out the `!!` as well then.